index.html

```diff
@@ -51,7 +51,7 @@
             
                   <a href="https://sef.discourse.group" class="text-gray-300 block px-3 py-2 text-base font-sm" target="_blank">SEF Hive</a>
             
-                  <a href="https://sef.discourse.group/top?period=yearly" class="inline-flex items-center justify-center font-sm transition-all px-4 py-2 text-center
+                  <a href="https://sef.discourse.group/top?period=yearly" class="inline-flex items-center justify-center font-sm px-4 py-2 text-center
```

Comment:
Here as well, you haven't replaced the link.

Reply:
replaced it too.